py2/SciServer/Jobs.py

```diff
@@ -408,17 +479,26 @@ def submitNotebookJob(notebookPath, dockerComputeDomain=None, dockerImageName=No
                     raise Exception("User volume '" + uVol.get('name') + "' not found within Compute domain")
 
         datVols = [];
-        if dataVolumes is None:
+        if dataVolumes == 'all':
             for vol in dockerComputeDomain.get('volumes'):
-                datVols.append({'name': vol.get('name')});
+                datVols.append({'id': vol.get('id'), 'name': vol.get('name'), 'writable': False});
```

Comment:
for uservolumes='all' the code decides whether to ask for 'write' based on privileges the user has.
why not do the same for data volumes